deploy/conf/babble/scripts/build-babble-conf.sh

```diff
@@ -24,7 +24,7 @@ do
 	echo "Generating key pair for node$i"
 	docker run \
 		-v $dest:/.babble \
-		--rm mosaicnetworks/babble keygen
+		--rm mosaicnetworks/babble keygen | sed -n -e "2 w $dest/key.pub" -e "4,+4 w $dest/priv_key.pem"
```

Comment:
Since this line not specific the babble image version, it will use the latest.
But the latest version is not v0.4.0.